about.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/b42bc5ce06.js" crossorigin="anonymous"></script>
+    <title>Capstone</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+        <!-- Header -->
+     <header id="header">
+        <div class="overlay about-overlay">
+            <img src="images/learn-code.jpeg" alt="">
+            <nav id="nav">
+                <div class="nav-inner"><div class="nav-child">
+                    <a href="index.html">
+                        <div class="logo" id="logo">
+                            <h3>My Logo</h3>
+                        </div>
+                    </a>
+                    <button id="bars">
+                        <i id="open" class="fas fa-bars"></i>
+                    </button>
+                </div>
+                <!-- Nav menu -->
+                <ul class="nav-menu">
+                    <li class="nav-menu-item"><a class="nav-link" href="index.html">home</a</li>
+                    <li class="nav-menu-item"><a class="nav-link nav-active" href="#about.html">About</a></li> 
+                </ul>
+
+                </div>
+                
+            </nav>
```

Comment:
- [ ] The navbar on the template is divided into two bars, one dark with the social media links, logout, and language links aligned to the right, and a white bar with the logo and links to the pages. Your navbar only has the bottom white bar. Kindly create the top dark bar to match the template on both pages.
 - Your design:
 - 
<img width="1080" alt="image" src="https://user-images.githubusercontent.com/61919617/217856928-e0fd35cb-4ddd-4a99-8e26-5078781b9dd6.png">

 - Template:
 - 
<img width="893" alt="image" src="https://user-images.githubusercontent.com/61919617/217856966-d2ecad0a-db1c-40e2-a74a-5b20f234ab94.png">